File: reminder.py
from kivy.utils import platform

from jnius import autoclass


def remind():
    if platform != 'android':
        raise NotImplementedError('Setting calendar reminders is only supported on Android')
    # noinspection PyPep8Naming
    Reminder = autoclass('edu.sepsis.cse.soft162.Reminder')
    success = Reminder.remind()
    if not success:
        raise NotImplementedError('Unable to find phone app')

File: edu/sepsis/cse/soft162/Reminder.java
package edu.sepsis.cse.soft162;

import org.kivy.android.PythonActivity;
import android.content.Intent;
import android.net.Uri;
import android.app.Activity;

public class Reminder {
    public static boolean remind() {
        Intent intent = new Intent(Intent.ACTION_INSERT);

        Activity activity = PythonActivity.mActivity;
        if (intent.resolveActivity(activity.getPackageManager()) != null) {
            activity.startActivity(intent);
            return true;
        }
        return false;
    }
}
